samples/Erlang/release.es

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env escript
```

Comment:
@pchaigno I was merely following the instructions laid out in the contribution notes:

> In addition, if your new language defines an extension that's already listed in `languages.yml` ... make sure that example .foo files are present in the samples directory for ***each*** language that uses `.foo`.

The Erlang directory lacked any files with an `.es` extension.